client/my-sites/email/titan-mail-quantity-selection/index.jsx

```diff
@@ -287,151 +221,141 @@ class TitanMailQuantitySelection extends React.Component {
 		);
 	}
 
-	renderPricingDetails() {
-		const { moment, selectedDomain, titanMonthlyProduct, translate } = this.props;
+	renderUnusedMailboxesNotice() {
+		const { maxTitanMailboxCount, selectedDomain, translate } = this.props;
+
+		if ( ! hasTitanMailWithUs( selectedDomain ) ) {
+			return null;
+		}
+
+		const numberOfUnusedMailboxes =
+			maxTitanMailboxCount - getConfiguredTitanMailboxCount( selectedDomain );
+
+		if ( numberOfUnusedMailboxes <= 0 ) {
+			return;
+		}
+
+		const text = translate(
+			'You have %(numberOfMailboxes)d unused mailbox. Do you want to configure it now instead?',
+			'You have %(numberOfMailboxes)d unused mailboxes. Do you want to configure them now instead?',
+			{
+				count: numberOfUnusedMailboxes,
+				args: {
+					numberOfMailboxes: numberOfUnusedMailboxes,
+				},
+				comment: 'This refers to the number of mailboxes purchased that have not been set up yet',
+			}
+		);
+
+		return (
+			<Notice icon="notice" showDismiss={ false } status="is-warning" text={ text }>
+				<NoticeAction
+					external={ ! isEnabled( 'titan/iframe-control-panel' ) }
+					onClick={ this.handleCreateMailbox }
+				>
+					{ translate( 'Finish Setup' ) }
+				</NoticeAction>
+			</Notice>
+		);
+	}
+
+	renderNewMailboxesNotice() {
+		const { moment, selectedDomain, translate } = this.props;
+
+		if ( ! hasTitanMailWithUs( selectedDomain ) ) {
+			return null;
+		}
 
-		const pricingTitle = <h3>{ translate( 'Pricing' ) }</h3>;
-		// Handle cases where we're only dealing with the standard price
 		if ( this.doesAdditionalPriceMatchStandardPrice() ) {
-			return (
```

Comment:
There is a problem here: if the pricing is standard, we are not showing the standard price or billing period on this page. I don't think we can drop that from the page, as we should be making it ultra-clear what the price and billing cycle are.

Reply:
Ah, right. I wanted to be consistent with G Suite where we don't display that information until checkout but it's probably better to show it as soon as possible 👍.